### DISCUSSION TITLE
Security is not enabled | Ingest manager

### DISCUSSION THREAD

**Discussion by rootsystemproton** [Security-Onion-Solutions/securityonion#1690] [Unsupported Versions] [answered]
Hi all,

I'm currently using security onion 2.3.0 and It's amazing by the way!
I've noticed the new feature Ingest manager that would simplify the process of adding data (metrics, filebeat etc), 
I'm having real problems following the guide docs/documentation as the location of things like elasticsearch-setup-passwords are not in the default Elastic location, this means I can't enable the security Kibana/Elastic requires for me to use the Ingest manager to add metrics and other data,

![image](https://user-images.githubusercontent.com/35872202/97374634-a666d580-18b0-11eb-87fa-1f8984816884.png)

Could someone please advise how I might go about getting this enabled so I have a more simple way of adding metrics?

I have tried to do this manually as I would have done with ELK installs over Debian but Elastic and Kibana don't seem to run on the CENTOS layer so the host isn't listening on its default ports on the given IP, so when I go to configure the following lines I'm unsure how I would connect the services;

output.elasticsearch:
  hosts: ["<es_url>"]
  username: "elastic"
  password: "<password>"
setup.kibana:
  host: "<kibana_url>"

Any help would be greatly appreciated.

I'm having a blast using the new Sec Onion, pulling in metrics and creating dashboards to visualise the SIEM events for the network, if I could get this checked off the list I'd be able to pull in host metrics of windows servers and hopefully other kit like Brocades and Fortigate firewalls as well.

Thanks all!

Kind regards

Luke

**Comment by weslambert** [comment, marked as answer]
I'm not sure we support Elastic Auth/Security just yet for Security Onion 2 -- will need to check on this.

**Reply by weslambert** [reply]
Confirmed we don't officially support Elastic auth or the configuration you described at the moment.
